boards/Pynq-Z1/Pynq-Z1.spec

```diff
@@ -2,4 +2,4 @@ ARCH_Pynq-Z1 := arm
 BSP_Pynq-Z1 := Pynq-Z1.bsp
 BITSTREAM_Pynq-Z1 := base/base.bit
 
-STAGE4_PACKAGES_Pynq-Z1 := pynq boot_leds ethernet
+STAGE4_PACKAGES_Pynq-Z1 := boot_leds ethernet
```

Comment:
We need these otherwise the board-specific overlays won't get installed in the image. STAGE3 is board-agnostic